internal/cli/atlas/accesslists/create.go

```diff
@@ -80,26 +83,85 @@ func (opts *CreateOpts) newProjectIPAccessList() []*atlas.ProjectIPAccessList {
 	return []*atlas.ProjectIPAccessList{entry}
 }
 
+func iPAddress() (string, error) {
```

Comment:
[nit] both UPPERCASE or both lowercase
```suggestion
func ipAddress() (string, error) {
```